practices/multiple_environments.md

```diff
@@ -0,0 +1,125 @@
+3. **Staging environment**
+   The staging environment or pre-production environment is an environment for testing that exactly resembles a production environment. It seeks to mirror an actual production environment as closely as possible and may connect to other production services and data, such as databases. 
+   The primary use of a staging environment is to test all the installation/configuration/migration scripts and procedures before they're applied to a production environment. This ensures all major and minor upgrades to a production environment are completed reliably, without errors, and in a minimum of time.
+   Another important use of staging is performance testing, particularly load testing, as this is often sensitive to the environment.
+   Staging environmet is also used by some organizations to preview new features to select customers or to validate integrations with live versions of external dependencies.
+3. **Production environment**
+   The production environment can also be said as a live environment, particularly for servers, as it is the environment that users directly interact with. Deploying to production is the most sensitive step; it may be done by deploying new code directly, or by deploying a configuration change.
+   Deploying a new release generally requires a restart, unless hot swapping is possible, and thus requires either an interruption in service (usual for user software, where applications are restarted) or redundancy – either restarting instances slowly behind a load balancer or starting up new servers ahead of time and then simply redirecting traffic to the new servers.
+   
+
+![Multiple Environments](/files/Multiple_Environments.png)</br>
+Source: [Using Multiple Environments to Improve Your Development Workflow](https://deploybot.com/blog/using-multiple-environments-to-improve-your-development-workflow)
+
+## Why Use Multi Environments
+
+Multi Environments helps you with:
+
+- **Keeps the team productive**
+  Having multiple environments enables a team to work on parallel development efforts. If there are several people working on the app, using a different environment for each group of people helps keep the team productive.
+- **It reduces or eliminates downtime and thus saves the company from loss of revenue**
+  Thoroughly testing software through multiple environments allows you to deliver a product to your users that is just as reliable and dependable as your company. If the software directly deals with the exchange of money, errors in the code could also lead to losses. 
+- **Provides better security**
+  To protect the integrity of your production data, you should limit access to it. Team members should have clearly defined roles and access rights to different parts of the system. Having multiple environments makes this possible. If you have separate development and production environments, it prevents developers from accidentally messing with or deleting production data.
```

Comment:
Please, give this a second thought. It has similarities to the original:

_To protect the integrity of your production data, you should limit access to it. Team members should have clearly defined roles and access rights to different parts of the system. Having multiple environments makes this possible. If you have separate development and production environments, it prevents developers from accidentally messing with or deleting production data_